hir/src/component/mod.rs

```diff
@@ -86,11 +84,14 @@ impl formatter::PrettyPrint for ComponentImport {
         };
         let name = match self {
             ComponentImport::CanonAbiImport(import) => import.interface_function.to_string(),
-            ComponentImport::MidenAbiImport(import) => import.function_id.to_string(),
+            ComponentImport::MidenAbiImport(_import) => "()".to_string(),
```

Comment:
Shouldn't this also be `import.interface_function.to_string()`? Or is there not a meaningful name in that field yet for this import type.

Reply:
Well, there is no any different public "outer" name (like the `WIT` one in CM) besides the one I already have in Wasm core module import (e.g. `miden:tx_kernel/note.get_inputs`) so I decided to omit it altogether and even removed the field for it from `MidenAbiImport`. Do you think we should keep it and print it here?